### ISSUE TITLE
Create a proper "Documentation" index page

### ISSUE THREAD

**Issue by JKAussieSkater** [gruntjs/grunt-docs#156]
Currently the "Documentation" link on Grunt's [website header](https://github.com/gruntjs/gruntjs.com/blob/master/src/tmpl/blocks/header.jade) links to the same location as "Getting Started".

It would be more practical to instead have the documentation link to a "documentation overview" landing page. Also, removing duplicated URLs is a good idea.

Feel free to comment to this issue if you have ideas on what a "Documentation Overview" landing page should look like.

**Comment by AurelioDeRosa** [comment]
I can take of this task if available but I'd like also to have some guidance. Is there anything that you definitely want to show on this page?

**Comment by AurelioDeRosa** [comment]
Friendly ping. I'd like to submit a PR today for your review.

**Comment by JKAussieSkater** [comment]
Oh awesome!
I look forward to pulling it!

**Comment by JKAussieSkater** [comment]
@AurelioDeRosa, I see you asked for some guidance, sorry I missed your call!

I saw [this GitHub one](https://git-scm.com/documentation), for a little inspiration, but it sounds like you've already made one!

**Comment by AurelioDeRosa** [comment]
Thank you for the suggestion. I've followed your hint to create the page.

**Comment by JKAussieSkater** [comment]
So far the introductory wording is fantastic; we just need to have a complete index of doc pages as well—like a "Contents" page for quick reference, and very much similar to the existing side-bars of the site.

**Comment by JKAussieSkater** [comment]
For everybody's reference, @AurelioDeRosa has created the initial [`Documentation.md`](https://github.com/gruntjs/grunt-docs/blob/master/Documentation.md) file.

**Comment by AurelioDeRosa** [comment]
Thank you for the shout out @JKAussieSkater. In regard of your point to improve the page, do you want to add the content of the sidebar in the Documentation page as well?

**Comment by JKAussieSkater** [comment]
Yeah! Maybe split it up into a two column grid just to see how it feels.

**Comment by AurelioDeRosa** [comment]
Do we have the possibility to split in columns using markdown? The only way without classes and html markup I can think of is to use a table which will lead to poor semantic.

**Comment by JKAussieSkater** [comment]
I don't think Git flavoured markdown (which can do tables) is supported by the Jade parser . . . I could be wrong.
But ultimately we need a good visual layout for an index, so we've got to do what we've got to do. HTML is the way to go.

**Comment by JKAussieSkater** [comment]
Using DIVs of course, not tables 😉

**Comment by dmethvin** [comment]
Looks like there is table support recently at least tho, I've used this tool:
http://www.tablesgenerator.com/markdown_tables

| Tables   |      Are      |  Cool |
|----------|:-------------:|------:|
| col 1 is |  left-aligned | $1600 |
| col 2 is |    centered   |   $12 |
| col 3 is | right-aligned |    $1 |

**Comment by AurelioDeRosa** [comment]
I have mixed sentiments on this matter. On one side I'd like to stick with Markdown and not starting the Markdown/HTML mix which will result in a mess. On the other hand, tables for semantic suck. If I'm forced to pick one, I'd rather go for Markdown/HTML mix though.

In regard of the classes to use, I've seen that the website uses Bootstrap v2. So, we should have a structure like:

```html
<div class="row-fluid">
   <div class="span6">
      Content here
   </div>
   <div class="span6">
      Content here
   </div>
</div>
```

Any further comment or are we good to go?

P.S.: Thank you @dmethvin!

**Comment by JKAussieSkater** [comment]
That's a sick generator @dmethvin!

@AurelioDeRosa, Let's go with the Bootstrap option.

**Comment by AurelioDeRosa** [comment]
I've submitted a PR based on our discussion. Hopefully it matches the expectations.

**Comment by JKAussieSkater** [comment]
@vladikoff, how might we go about using HTML in our *.md files? It doesn't seem to be compatible with the Jade parser on the stage server:
![Raw HTML when parsing *md files](https://cloud.githubusercontent.com/assets/12983913/13560117/ffb4d9ae-e46f-11e5-8a83-99f0dbaa3304.png)

**Comment by AurelioDeRosa** [comment]
Perhaps there is an escape HTML entities in place? For instance, in Handlebars instead of using `{{variable}}` you have to write `{{{variable}}}`.

**Comment by JKAussieSkater** [comment]
Looks like vladikoff is busy this week, @shama, do you have any ideas how we can embed HTML in the Jade/Markdown files?

**Comment by steveoh** [comment]
escape as it show html?

```html
<div>Like this</div>
```

Is the markdown processing different with the gh-pages? If it's not, then I would think you could use kramdown syntax and the code highlighting would be rouge. If this were the case, you can apply css to markdown items yada yada.

**Comment by shama** [comment]
@JKAussieSkater I'm not too familiar but the docs are built with gollum correct? So depending on the tags, using HTML in the markdown should work: https://github.com/gollum/gollum/wiki/Security#page-sanitization. As for Jade, they have syntax for raw HTML: http://jade-lang.com/reference/plain-text/

**Comment by AurelioDeRosa** [comment]
Is there any update for the HTML code of the documentation that was removed for the release?
